eng/emitters/pipelines/templates/stages/emitter-stages.yml

```diff
@@ -104,6 +108,7 @@ stages:
           NodeVersion: 20.x
           Os: linux
           EmitArtifacts: true # Emit artifacts only for the first job
+          PythonVersion: ${{ parameters.PythonVersion }}
```

Comment:
According to CI [log](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4710201&view=logs&jobId=21ade72b-9422-585e-265b-4ef953b62f93&j=6f108f5e-3d2f-53d4-8312-e1c4a6db7ff7&t=8b614e3a-2869-5755-bf1e-dd02dbf673f6), default python version for linux is 3.8 which has been end of life.
![image](https://github.com/user-attachments/assets/8b418229-e6d2-4fd0-aaec-9fbac8962be8)